lib/main.dart

```diff
@@ -26,9 +26,9 @@ class MyApp extends StatelessWidget {
       ),
       initialRoute: '/',
       routes: {
-        '/': (context) => MyHomeRoute(title: 'GitHub Client dos Guri'),
```

Comment:
Why do you decide to add type annotations for variables/parameters?
If Flutter provides Type Inference feature and this makes code more simpler.

Reply:
Because i consider it better for code interpretation, even when the language infer the type. When you are reading the code you can say what it is more easily